scripts/fuzz_opt.py

```diff
@@ -699,7 +696,6 @@ def randomize_opt_flags():
 # contains the list of all possible feature flags we can disable (after
 # we enable all before that in the constant options)
 POSSIBLE_FEATURE_OPTS = run([in_bin('wasm-opt'), '--print-features', '-all', in_binaryen('test', 'hello_world.wat'), '-all']).replace('--enable', '--disable').strip().split('\n')
```

Comment:
We should probably replace `-all` here with `CONSTANT_FEATURE_OPTS` to avoid duplicate flags, too.

Reply:
Done.